apis/v1alpha1/validate.go

```diff
@@ -132,28 +132,20 @@ func (in *GreptimeDBCluster) validateFrontend() error {
 }
 
 func (in *GreptimeDBCluster) validateFrontendGroups() error {
-	nameCount := 0
+	count := 0
 	for _, frontend := range in.GetFrontendGroups() {
 		if len(frontend.GetName()) != 0 {
```

Comment:
I think it should return error directly if the name is empty.

Reply:
like this:
```go
if len(frontend.GetName()) == 0 {
  return fmt.Errorf("the frontend name must be specified")
}
```